dask/dataframe/indexing.py

```diff
@@ -10,7 +10,7 @@
 from ..highlevelgraph import HighLevelGraph
 from . import methods
 from ._compat import PANDAS_GT_130
-from .core import Series, new_dd_object
+from .core import Series, is_bool_dtype, new_dd_object
```

Comment:
The import here should be from `pandas.api.types.is_bool_dtype`. That's where `core.py` imports it from.